websites/G/Gartic.io/presence.ts

```diff
@@ -1,38 +1,41 @@
-const presence = new Presence({
-		clientId: "808668919635247104",
-	}),
-	browsingTimestamp = Math.floor(Date.now() / 1000);
+/* eslint-disable one-var */
+const garticPlayingTimestamp = Math.floor(Date.now() / 1000);
+const getGarticPresence = (): PresenceData => {
+	const path = document.location.pathname,
+		base: PresenceData = {
+			largeImageKey: "https://i.imgur.com/JkOc5Vo.png",
+			startTimestamp: garticPlayingTimestamp,
+		};
+
+	// The homepage
```

Comment:
These kinds of comments are a bit redundant based on the code.
```suggestion
```

Reply:
Resolved in https://github.com/PreMiD/Presences/pull/7158/commits/8c42b9f28faf672356e3dbc401e30f6529b0bc3a